contribute.md

```diff
@@ -97,48 +97,52 @@ In general, any Ruby version supported by Rails version required by Foreman will
 
 The Rails version is found by opening the [Gemfile](https://github.com/theforeman/foreman/blob/develop/Gemfile) and searching for `gem 'rails'` line. Then find out which Ruby versions are supported for that particular Ruby on Rails version. Another, perhaps faster way, is to visit our [Jenkins CI server](http://ci.theforeman.org/view/Foreman%20pipeline/job/test_develop) to see matrix of supported versions. Then pick any version from that list.
 
-Instructions for Fedora, CentOS 8, CentOS 8 Stream or Red Hat Enterprise Linux 8:
+Instructions for Enterprise Linux (Almalinux, CentOS Stream, Red Hat Enterprise Linux, Rocky Linux) 9:
 
 ```
-dnf module enable ruby:2.7
+# (OPTIONAL) Change Ruby module version before installing
+dnf module enable ruby:3.3
+# (OPTIONAL) Switch Ruby module version if packages are already installed
+dnf module switch-to ruby:3.3
+# Install ruby packages
 dnf install ruby ruby-devel
 ```
 
 #### Supported Node.js versions
 
 In general, the latest stable Node.js should work. However if you are just starting out, you might want to develop against one of the versions that we test against to reduce the chances of you hitting an unexpected issue, although this is not required.
 
-You can view our [Github Actions configuration](https://github.com/theforeman/foreman/blob/develop/.github/workflows/js_tests.yml) to see supported NodeJS versions. Any version from that list will do. For populating the node_modules folder, npm version 4 or later is required. Alternatively, [yarn](https://yarnpkg.com) can also be used.
+You can view our [package.json configuration](https://github.com/theforeman/foreman/blob/develop/package.json) to see supported NodeJS versions. Any version from that list will do. For populating the node_modules folder, npm version 4 or later is required. Alternatively, [yarn](https://yarnpkg.com) can also be used.
 
-Instructions for Fedora, CentOS 8, CentOS 8 Stream or Red Hat Enterprise Linux:
+Instructions for Enterprise Linux 9:
 
 ```
-dnf module enable nodejs:14
+# (OPTIONAL) Change Ruby module version before installing
+dnf module enable nodejs:18
+# (OPTIONAL) Switch Ruby module version if packages are already installed
+dnf module switch-to nodejs:18
+# Install Node.js
 dnf install nodejs
 ```
 
 #### Supported PostgreSQL versions
 
-In general, the latest version of PostgreSQL will work. Minimum version is currently 10, anything newer than that will do the job.
+Foreman requires PostgreSQL, even for development. SQLite is no longer supported and will not work. In general, the latest version of PostgreSQL will work. Minimum version is currently 13, anything newer than that will do the job.
```

Comment:
We haven't supported SQLite since Foreman 2.0. Perhaps we can leave that out by now.

Reply:
Ok, 5 years it's enough, removed it :)